src/NumericalAlgorithms/DiscontinuousGalerkin/MortarHelpers.hpp

```diff
@@ -82,6 +89,12 @@ Variables<Tags> project_from_mortar(
     const Mesh<Dim>& face_mesh,
     const Mesh<Dim>& mortar_mesh,
     const std::array<Spectral::MortarSize, Dim>& mortar_size) noexcept {
+  if (face_mesh == mortar_mesh and
+      alg::all_of(mortar_size, [](const Spectral::MortarSize& size) noexcept {
```

Comment:
Due to program logic this is never true (except in tests, probably).  This check is done at the call site because there we can avoid the copy.

Reply:
Okay, sounds good. Would you be okay with me putting in an ASSERT that states what you just said? That way in case someone changes the code and it can't accidentally slip by us in code review :)

Reply:
Okay, I'm pushing with the `ASSERT` but am happy to remove it if you don't think it's a good thing to have.

Reply:
Yeah, that seems fine.  If someone runs into a case where they really want to project from a mortar and then use the data again they can remove it, but we probably won't want that. (Might have been useful if we'd stuck with interface chares.)